### DISCUSSION TITLE
Hybrid ServerType - use Plex and Jellyfin concurrently

### DISCUSSION THREAD

**Discussion by Xuroth** [seerr-team/seerr#305] [General]
I am curious if there is a flag/setting/environment variable that can be enabled (via settings file or sqlite db query, et al) to allow Jellyseerr to run two media servers simultaneously?

 My usecase is that I am actively homelab-ing and would like to provide Jellyfin and Plex servers for different clients (some televisions do not have Jellyfin as an app 😢 , but do support Plex). I'm in the process of trying to use Jellyfin as my primary server, and Plex as a fallback. I currently have both servers running, but I was curious if it was possible to add a `jellyfin` key to /settings.json (with proper values) or something.
 
 If not, I suppose the workaround would be to host two instances of Jellyseerr, each dedicated to the respective server?

**Comment by fallenbagel** [comment]
Simultaneous concurrent server setup is in the roadmap but not possible yet. The way to achieve this would be to run two instances with different ports. I recommend running overseerr for plex and jellyseerr for jellyfin

**Reply by Xuroth** [reply]
I appreciate the reply, thanks! I'll go ahead and set up two separate containers. I use Traefik, so I should (in theory) have no conflicts.

May I ask why you recommend using overseerr for plex instead of a dedicated Jellyseerr instance?

**Reply by fallenbagel** [reply]
> May I ask why you recommend using overseerr for plex instead of a dedicated Jellyseerr instance?

Jellyseerr is a fork of overseerr with focus on jellyfin/emby. That does mean that we pull in upstream (overseerr) commits so plex fully works the same as it does on overseerr, HOWEVER, this upstream pull is done manually so overseerr would always be more updated when it comes to plex. As I work on this on my free time as well, sometimes upstream merges might take a week or two or even three. 

That being said, it's perfectly fine to use jellyseerr too, so it's mainly up to you to decide c:

**Reply by fallenbagel** [reply]
Oh and in addition, Overseerr support could give better plex related support than me since I do not run plex (that being said once again, if its a core issue and not a plex server side issue [tho i could probably help resolve this too if im familiar with the issue that is being raised since i did use plex for a long time as well], I can easily provide support as well)

**Reply by Xuroth** [reply]
Thanks for the response!

That makes perfect sense, and is more or less what I was thinking of. I'll continue to use Jellyseerr for Jellyfin, and I'll use Overseerr for Plex requests. Now my big step is figuring out to unify accounts and access (as much as possible, at least) so less technical users in my smarthome can use it with minimal fuss (but that's kinda the fun part, making everything work together)

I appreciate your time in responding, as well as your efforts on this project!

**Comment by XDr46on** [comment]
Is it possible to use Plex and Jellyfin at the same time now that the projects have merged to form Seer, or is that still on the roadmap?

**Reply by fallenbagel** [reply]
Still on roadmap. Now you'll have to run two seerr instances as a workaround
